browser/ts/index.ts

```diff
@@ -97,6 +97,14 @@ export interface Config {
     // 30 rps for Infura's free tier, and can be increased to 100 rpc for pro
     // users, and potentially higher on alternative infrastructure.
     ethereumRPCMaxRequestsPerSecond?: number;
+    // Determines whether or not Mesh should limit the number of Ethereum RPC
+    // requests it sends. It defaults to true. Disabling Ethereum RPC rate
+    // limiting can reduce latency for receiving order events in some network
+    // conditions, but can also potentially lead to higher costs or other rate
+    // limiting issues outside of Mesh, depending on your Ethereum RPC provider.
+    // If set to false, ethereumRPCMaxRequestsPer24HrUTC and
+    // ethereumRPCMaxRequestsPerSecond will have no effect.
+    enableEthereumRPCRateLimiting?: boolean;
```

Comment:
Can we move this config _above_ the 24hr and perSecond configs? Here and in the other files too.

Reply:
👍Good idea.